LabTools/CoreWidgets/InstrumentWidget.py

```diff
@@ -592,7 +592,7 @@ def bt_connecthub_clicked(self):
         in the main window"""
         if self.bt_connecthub.isEnabled:
 
-            self.bt_connecthub.setStyleSheet("background-color : '#b3e0ff'")
+            # HANDLED IN ACTUAL CONNECT FUNCTION: self.bt_connectHub_clickedstyle()
```

Comment:
Why leaving this here? We have a version control if we want to see how it was done beforehand :)

Reply:
I left that here as a note for anyone in the future, to indicate which is in the connection function